DESCRIPTION.md

```diff
@@ -10,7 +10,6 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v2.8.3(Unreleased)
```

Comment:
```suggestion
- v2.8.3(November 28, 2022)
```

Reply:
I forgot to push this, thanks for catching it!